### DISCUSSION TITLE
Ponderacion pruebas

### DISCUSSION THREAD

**Discussion by ignaciomanzanares** [IIC2133-PUC/2024-1#144] [Ayuda] [answered]
Tengo una duda sobre las ponderaciones de las pruebas, ya que en el programa no especifica cuanto vale cada una. Ademas, multiplica la i2 y la i3 con la ponderacion i2, es un error eso?

**Comment by MaxAl100** [comment, marked as answer]
1°: https://github.com/orgs/IIC2133-PUC/discussions/65 (xd)
2°: En el programa dentro de la carpeta Administrativo está: https://github.com/IIC2133-PUC/2024-1/blob/main/Administrativo/programa.pdf, donde se puede ver que la I1 y la I2 valen lo mismo, un 35% de la nota promedio de interrogaciones, mientras que la I3 vale un 30%.
